fabtests/scripts/runmultinode.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+
+echo "@=$@"
+
+Options=$(getopt --options h:,n:,p:,I:,C: \
+		  		--longoptions hosts:,processes-per-node:,provider,pattern:,iterations:,cleanup \
+				-- "$@")
+
+eval set -- "$Options"
+
+hosts=[]
+ppn=1
+iterations=1
+pattern=""
+cleanup=false
+
+while true; do
+	case "$1" in
+		-h|--hosts)
+			IFS=',' read -r -a hosts <<< "$2"; shift 2 ;;
+		-n|--processes-per-node) 
+			ppn=$2; shift 2 ;;
+		-p|--provider)
+			provider="$2"; shift 2 ;;
+		-I|--iterations)
+			iterations=$2; shift 2 ;;
+		-C|--pattern)
+			pattern="$2"; shift 2 ;;
+		--cleanup)
+			cleanup=true; shift ;;
+		--)
+			shift; break ;;
+	esac
+done
+		
+num_hosts=${#hosts[@]}
+ranks=$(($num_hosts*$ppn))
+server=${hosts[0]}
+start_server=0
+output="multinode_server_$ranks.out"
+
+cmd="fi_multinode -n $ranks -s $server -p '$provider' -C $pattern -I $iterations"
+echo $cmd
+
+if ! $cleanup ; then
+  
+	for node in "${hosts[@]}"; do
+		for i in $(seq 1 $ppn); do
+			if [ $start_server -eq 0 ]; then
```

Comment:
I am not sure why "start_server" eq 0 means start server? Should not it be the other way (e.g. `start_server -eq `` means start server?